src/AbstractJob.php

```diff
@@ -143,7 +144,7 @@ public function __toString()
     /**
      * @see ArrayAccess::offsetSet()
      */
-    public function offsetSet($offset, $value)
+    public function offsetSet(mixed $offset, mixed $value): void
```

Comment:
What does PHP 7.3 think about this type hint (as the composer.json says we're still supporting it)?